microhapulator/pipeaux.py

```diff
@@ -187,27 +187,27 @@ def read_length_table_single_end(samples):
 
 def aggregate_plots_paired_end(samples):
     plots = {
-        "r1readlen": list(),
-        "r2readlen": list(),
-        "mergedreadlen": list(),
+        "r1readlen": str,
+        "r2readlen": str,
+        "mergedreadlen": str,
```

Comment:
These are intended as placeholders, not type hints, so an empty string would be a good default.